docs/USER_RECORD.md

```diff
@@ -314,7 +314,20 @@ user. When logging in
 [`pam_systemd`](https://www.freedesktop.org/software/systemd/man/pam_systemd.html)
 will automatically initialize the `$LANG` environment variable from this
 string. The string hence should be in a format compatible with this environment
-variable, for example: `de_DE.UTF8`.
+variable, for example: `de_DE.UTF-8`. You should use `languages` instead.
+
+`languages` → An array of strings indicating the preferred languages/locales for
+the user, listed in order of descending priority. This allows multi-lingual users
+to specify all the languages that they know, so software lacking translations in
+the user's primary language can fall back to another language that the user knows
+rather than the default English. When logging in
+[`pam_systemd`](https://www.freedesktop.org/software/systemd/man/pam_systemd.html)
+will automatically initialize the `$LANGUAGE` variable from this list. `pam_systemd`
+will also initialize the `$LANG` environment variable with the first entry in this
+array (i.e. the user's primary language), ignoring `preferredLanauge`. All the entires
```

Comment:
"Lanauge"?

Reply:
I think I like preferred more, since the intent is for desktop environments to change the setting every time you log in with a new desktop

Reply:
> moreover we actually have a concept for this already in logind's session "desktop" field, and the XDG_SESSION_DESKTOP env var.

Looking at the docs, I don't think this is the same thing

`sessionName` is the name of a file that matches `/usr/share/{wayland-,x}sessions/*.desktop`. So if `sessionName` is set to `gnome`, then GDM will look at and load `/usr/share/wayland-sessions/gnome.desktop` to find out what it needs to run to start GNOME. If it is set to `gnome-x11`, then GDM will try to load `/usr/share/wayland-sessions/gnome-x11.desktop`, find it missing, and eventually settle on `/usr/share/xsessions/gnome-x11.desktop`

These names are notably _not_ the same as the XDG_CURRENT_DESKTOP, which are used to identify the DE but not the specific session .desktop file used to start it. And the docs say that XDG_SESSION_DESKTOP should pretty much be equivalent to XDG_CURRENT_DESKTOP's first entry